### PR TITLE
Jp dashboard

### LINKED ISSUE #92: View Rc (API)

### PR DESCRIPTION
**Pull request by amdpt323**
- closes #92

### CODE REVIEW

File: pages/admin/rc/[rcid]/proforma/[proformaId]/index.tsx [hunk 1]
```diff
@@ -1,148 +1,151 @@
-import Meta from "@components/Meta";
-import StepperComp from "@components/Stepper/stepperComp";
-import { Card, Grid, Stack, TextField } from "@mui/material";
+import { Button, Card, Grid, Stack, Typography } from "@mui/material";
```
Comment (ManasGupta1310) [inline pages/admin/rc/[rcid]/proforma/[proformaId]/index.tsx:1]: This file already exists in main repo and I had changed its responsiveness. Please do not commit this file

File: pages/admin/rc/index.tsx [hunk 2]
```diff
@@ -54,6 +55,16 @@ const rows = [
 
 function Index() {
   const router = useRouter();
+  const [setRc] = useState();
```
Comment (ManasGupta1310) [inline pages/admin/rc/index.tsx:58]: Change this setRc to setRCList

File: pages/admin/rc/index.tsx [hunk 3]
```diff
@@ -54,6 +55,16 @@ const rows = [
...
+
+  useEffect(() => {
+    const response = async () => {
+      const some_var = await admin_getrc();
```
Comment (ManasGupta1310) [inline pages/admin/rc/index.tsx:62]: rename some_var to RCs

**Review by ManasGupta1310 — CHANGES_REQUESTED** [review]

**Comment by 1-Harshit** [comment]
What's going on here?

**Comment by 1-Harshit** [comment]
closing, inactivity

**Comment by 1-Harshit** [comment]
Closed

**Pull request closed without merging**